src/testplane.ts

```diff
@@ -119,6 +152,25 @@ export class Testplane extends BaseTestplane {
         return !this.isFailed();
     }
 
+    protected async _readFailed(): Promise<FailedListItem[]> {
+        try {
+            const data = await fs.readFile(this._config.failedTestsPath, "utf8");
+            return JSON.parse(data);
+        } catch {
+            return [];
+        }
+    }
+
+    protected async _saveFailed(): Promise<void> {
+        const dirname = path.dirname(this._config.failedTestsPath);
```

Comment:
Thinking about using this in CI, I think it would be convenient to be able to fetch this file from remote URL...

But that'd require to have two separate options for input and output. Generally, let's discuss with the team if we need run options like grep and files to be in config?
This would make possible doing so:

```js
export default {
    grep: /some-test/,
    lastFailed: {
        only: true,
        output: '.testplane/file.json',
        input: 'https://ci-artifacts.com/.../file.json',
    }, // or just true/false if default input/output are sufficient

    // other config options
}
```